TFT/src/User/API/boot.h

```diff
@@ -143,10 +143,11 @@ enum
   ICON_LEDCOLOR,
   
 //  ICON_RESERVE
-// Preview should be in the last place
-  ICON_PREVIEW,
 // Back ground sign
   ICON_BACKGROUND,
+
+  // Preview should be in the last place
+  ICON_PREVIEW,

```

Comment:
putting ICON_PREVIEW after ICON_BACKGROUND is just wastage of flash storage as the ICON_BACKGROUND is not used to write anything on the flash storage.
placing the ICON_PREVIEW after ICON_BACKGROUND just skips 1 icon count wasting the flash space at address of ICON_BACKGROUND.